doc/source/ray-more-libs/joblib.rst

```diff
@@ -6,9 +6,8 @@ Distributed Scikit-learn / Joblib
 .. _`issue on GitHub`: https://github.com/ray-project/ray/issues
 
 Ray supports running distributed `scikit-learn`_ programs by
-implementing a Ray backend for `joblib`_ using `Ray Actors <actors.html>`__
-instead of local processes. This makes it easy to scale existing applications
-that use scikit-learn from a single node to a cluster.
+implementing a Ray backend for `joblib`_ using `Ray Actors <../ray-core/actors.html>`__
+instead of local processes. This makes it easy to distribute existing application process that use scikit-learn from a single node to a cluster.
```

Comment:
I prefer the existing wording here, I think talking about processes is too low level for this description. if we want to add it, we can add a more detailed "architecture" section